code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -51,6 +52,8 @@
 	origin_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 5, TECH_POWER = 3)
 	slot_flags = SLOT_BELT
 	force = 8 //looks heavier than a pistol
+	w_class = 4	//Looks bigger than a pistol, too.
+	fire_delay = 6	//This one's not a handgun, it should have the same fire delay as everything else

```

Comment:
Normally this would re-balance it in exchange for its weight change, but given its infinite ammo, this seems like a poor decision.